### ISSUE TITLE
Add checker for unnecessary ellipsis

### ISSUE THREAD

**Issue by mbyrnepr2** [PyCQA/pylint#5460]
### Current problem

Add a checker for unnecessary ellipsis.

```
def my_function():
    """docstring & elipses"""
    ...  # nothing emitted

def my_second_function():
    """docstring & pass"""
    pass  # pylint emits: `my_test.py:7:4: W0107: Unnecessary pass statement (unnecessary-pass)`
```

### Desired solution

Emit a warning for this scenario, similar to when a docstring is immediately followed by `pass`.

**Comment by Pierre-Sassoulas** [comment]
If we implement this the documentation should specify that it's not required when using black.

**Comment by mbyrnepr2** [comment]
@Pierre-Sassoulas. Thank you for checking this. I don't understand the above comment about how it relates to black, perhaps I am missing something obvious here. Thanks!

**Comment by Pierre-Sassoulas** [comment]
Ho sorry I made a mistake and it was not very clear. We have a section in the doc where [we tell user what message can be disabled when using other linter](https://pylint.pycqa.org/en/latest/faq.html#i-am-using-another-popular-linter-alongside-pylint-which-messages-should-i-disable-to-avoid-duplicates). I thought black removed unnecessary ellipsis when in fact it does not: I just checked. So this is useful even when using black.
